makefile

```diff
@@ -16,12 +16,9 @@ list: # PRIVATE
 # *********************** SETUP ***********************
 
 # Install all 3rd party dependencies.
-install: check-node
+install: check-node check-yarn
 	@echo 'Installing 3rd party dependencies…'
-	@npm install
-	@echo '…done.'
-	@echo 'Removing any unused 3rd party dependencies…'
-	@npm prune
```

Comment:
yeah i was going to `yarn clean` but their specific warning gave me pause...

if it becomes an issue, `make reinstall` should cover you? `prune` was always just a courtesy really anyway, to my mind